molecule/update_cartridge/molecule.yml

```diff
@@ -49,6 +49,7 @@ scenario:
     - syntax
     - create
     - converge
+    # - idempotence
```

Comment:
Debug?

Reply:
No, it's to show that this step is skipped here